templates/generic_show_form.html.twig

```diff
@@ -134,21 +134,40 @@
                         ) }}
                      {% endif %}
 
+
+                     {% set conditions_met = 0 %}
                      {% set dc_breadcrumbs %}
                         {% if item is usingtrait('Glpi\\Features\\DCBreadcrumb') %}
                            {{ call(item.getType() ~ '::renderDcBreadcrumb', [
                               item.getID
                            ])|raw }}
                         {% endif %}
                      {% endset %}
-                     {% if dc_breadcrumbs|trim|length > 0 %}
+                    {% if dc_breadcrumbs|trim|length > 0 %}
                         {{ fields.htmlField(
-                           '',
-                           dc_breadcrumbs,
-                           __('Data center position'),
-                        ) }}
-                        {{ fields.nullField() }} {# set an empty space in front of dc breadcrumb #}
-                     {% endif %}
+                        '',
+                        dc_breadcrumbs,
+                        __('Data center position'),
+                        ) }}
+                        {% set conditions_met = conditions_met + 1 %}
+                    {% endif %}
+
+                    {% set cluster = null %}
+                    {% if item_type == 'Computer' or item_type == 'NetworkEquipment' %}
```

Comment:
This should probably rely on `$CFG_GLPI['cluster_types']`; and this logic should not be handled on view side.

Reply:
👍